site/static/style.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
I don't know that this is strictly correct or strictly necessary, but as far as I can tell, it's producing a correct-looking site.